extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -396,6 +419,208 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 	}
 	//#endregion
 
+	//#region Database Properties - Files Tab
+	private initializeFilesGeneralSection(): azdata.GroupContainer {
+		let containers: azdata.Component[] = [];
+		// Database name
+		this.nameInput = this.createInputBox(async () => { }, {
+			ariaLabel: localizedConstants.DatabaseNameText,
+			inputType: 'text',
+			enabled: this.options.isNewObject,
+			value: this.objectInfo.name
+		});
+		containers.push(this.createLabelInputContainer(localizedConstants.DatabaseNameText, this.nameInput));
+
+		// Owner
+		let loginNames = this.viewInfo.loginNames?.options;
+
+		if (loginNames?.length > 0) {
+			// Removing <default> login name from the list and adding current owner if not exists
+			if (!this.viewInfo.loginNames?.options.find(owner => owner === this.objectInfo.owner)) {
+				loginNames[0] = this.objectInfo.owner;
+			} else {
+				loginNames.splice(0, 1);
+			}
+			let ownerDropbox = this.createDropdown(localizedConstants.OwnerText, async () => {
+				this.objectInfo.owner = ownerDropbox.value as string;
+			}, loginNames, this.objectInfo.owner);
+			containers.push(this.createLabelInputContainer(localizedConstants.OwnerText, ownerDropbox));
+		}
+
+		// This check box is checked and disabled because full-text indexing is always enabled in SQL Server
```

Comment:
Why do we have this then? @erinstellato-ms

Reply:
Good catch @Charles-Gagnon.  @ssreerama Full-text indexing is not ALWAYS enabled in SQL.  It is optional on the install for an on-prem SQL Server instance.  It may always be enabled for MI and Azure SQL...  Let me know if you need help verifying that (I would like SMO would pick it up?).

Reply:
I verified for couple of servers SQL and MI and this is enabled for SQL Server(on-prem)(same mentioned in the help doc)  and disabled for MI. 
In SSMS there is no option available for MI as we don's show Files tab for MI. 

I'm sending the value for SQL server to enable files tab, and null for other instances to not show the files/filegroups tabs. @erinstellato-ms can you verify and let me know if there is any other place that we modify this property? Thanks

Reply:
@ssreerama Well, confirmed in the docs (https://learn.microsoft.com/sql/relational-databases/databases/database-properties-files-page) that the full-text indexing box is checked and disabled because full-text indexing is always enabled in SQL Server. That said, full text is an optional component of the engine that you have to select when you install SQL.

Frankly, this check box makes no sense.  To @Charles-Gagnon's point, I don't know why we need to have it.  If the user can never do anything with it, it's just taking up space.  I'm ok with removing it.